docs/apis/subsystems/files/index.md

```diff
@@ -293,6 +293,19 @@ $fs = get_file_storage();
 $fs->create_file_from_storedfile($fileinfo, $existingfile);
 ```
 
+#### Notification requirements for new files
+
+By default, the `$notify` parameter has a value of `true` and will call the after_file_created hook after files are created. If you prefer not to do this, you can set it to `false`.
```

Comment:
This is very unclear. What parameter? Of which function? What are we trying to achieve? I would mentioned the function directly and explained a new argument meaning - thery is no connection between argument name notufy and the fact that we are calling come hook. And same appies to the section name - what requirement is meant?